BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+    from pathlib import Path
+
+    def start_ipfs_daemon():
+        """
+        Open a new terminal and run the `ipfs daemon` command in the user's home directory.
+        """
+        home_dir = os.path.expanduser("~")  # Get user's home directory
+
+        try:
+            if sys.platform == "win32":
+                # Windows: Use `start` to launch cmd and set working directory
+                subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)